src/catkin_pkg/package_templates.py

```diff
@@ -409,13 +427,54 @@ def get_person_tag(tagname, person):
                     dep.version_lte,
                     dep.version_gt,
                     dep.version_gte
-                    )
+                )
                 dependencies.append(dep_tag)
             else:
                 dependencies.append(_create_depend_tag(dep_type,
                                                        dep.name))
     temp_dict['dependencies'] = ''.join(dependencies)
 
+    depends_dep_type = []
+    for dep in sorted(dep_map['run_depend'], key=lambda x: x.name):
```

Comment:
Instead of using a lambda please use a list comprehension:

```for dep in sorted([d.name for d in dep_map['run_depend']]):```

Same a few times below.